src/modules/powerrename/PowerRenameContextMenu/PowerRenameContextMenu.vcxproj.filters

```diff
@@ -242,6 +242,24 @@
     <None Include="Assets\**">
       <Filter>Resource Files</Filter>
     </None>
+    <None Include="Assets\**">
```

Comment:
feels like a dup, what caused this?

Reply:
there feels like a lot of dups added

Reply:
Visual Studio 2022 added these automatically. I can remove them